src/engine/helper/pka_helper.h

```diff
@@ -112,6 +112,35 @@ int pka_rsa_mod_exp_crt(pka_bignum_t *bn_value,
                         pka_bignum_t *bn_qinv,
                         pka_bignum_t *bn_result);
 
+// This functions implements the elliptic curve point addition using
+// Bluefield PKA hardware.
+int pka_ecc_bn_pt_add(pka_bignum_t *bn_p,
```

Comment:
pka_bn_ecc_pt_add() to be consistent with the above.

Reply:
Ack, Thanks.